src/parsers/pm_io/psse.jl

```diff
@@ -1223,6 +1223,62 @@ function _psse2pm_dcline!(pm_data::Dict, pti_data::Dict, import_all::Bool)
     end
 end
 
+function _psse2pm_facts!(pm_data::Dict, pti_data::Dict, import_all::Bool)
+    @info "Parsing PSS(R)E FACTs devices data into a PowerModels Dict..."
+    pm_data["facts"] = []
+
+    if haskey(pti_data, "FACTS CONTROL DEVICE")
+        for facts in pti_data["FACTS CONTROL DEVICE"]
+            @info(
+                """FACTs are supported via a simplification approach for terminal_bus = 0 (STATCOM operation)"""
+            )
+            sub_data = Dict{String, Any}()
+
+            sub_data["name"] = facts["NAME"]
+            sub_data["control_mode"] = facts["MODE"]
+
+            # MODE = 0 -> Unavailable
+            # MODE = 1 -> Normal mode 
+            # MODE = 2 -> Link bypassed
+            if facts["MODE"] != 0
+                sub_data["available"] = 1
+            else
+                sub_data["available"] = 0
+            end
+
+            sub_data["bus"] = facts["I"] # Sending bus number
+
+            if facts["J"] != 0
+                @warn "Series FACTs not supported."
+            end
+
+            if facts["MODE"] > 3
+                throw(DataFormatError("Operation mode not supported."))
+            end
```

Comment:
```suggestion
            if facts["J"] != 0
                @warn "Series FACTs not supported."
            end

            if facts["MODE"] > 3
                throw(DataFormatError("Operation mode not supported."))
            end
```
We should throw these warnings and error in the `make_facts` function call so that the parser's scope is limited to parsing as much as possible. I'd rather fail in the `make_facts` function such that if we ever need to support it we tackle it there.